Cargo.toml

```diff
@@ -105,7 +102,7 @@ panic = "abort"
 [package.metadata.deb]
 assets = [
     ["target/release/oxipng", "usr/bin/", "755"],
-    ["target/release/assets/oxipng.1", "usr/share/man/man1/", "644"],
+    ["target/release/manpages/oxipng.1", "usr/share/man/man1/", "644"],
```

Comment:
If I'm understanding everything correctly, this will only work within the deploy workflow, right? If you just run `cargo deb` normally the path will surely not be correct. Will it work if you put the xtask path here and skip the move in the workflow?

Reply:
You're right! I think it's a good idea to skip the move in the workflow, that way there are less moving parts in this automation :+1: